contracts/simple_dex/lib.rs

```diff
@@ -268,46 +262,6 @@ mod simple_dex {
             Ok(())
         }
 
-        /// Liquidity deposit
-        ///
-        /// Can only be performed by an account with a LiquidityProvider role
-        /// Caller needs to give at least the passed amount of allowance to the contract to spend the deposited tokens on his behalf
-        /// prior to executing this tx
-        #[ink(message)]
-        pub fn deposit(&mut self, deposits: Vec<(AccountId, Balance)>) -> Result<(), DexError> {
```

Comment:
why do we remove depositing?

Reply:
this is an artifact from when we still floated the idea of having a full-featured DEX with liquidity provider, but now this will never be used - only A0 liquidity will ever be added and this can be done with a simple PSP22 transfer to the contracts account